common/predictive-text/unit_tests/headless/worker-initialization.js

```diff
@@ -12,23 +12,30 @@ describe('LMLayerWorker', function() {
   describe('#constructor()', function() {
     it('should allow for the mocking of postMessage()', function () {
       var fakePostMessage = sinon.fake();
-      var worker = new LMLayerWorker({ postMessage: fakePostMessage });
+      var context = {
+        postMessage: fakePostMessage
+      };
+      context.importScripts = importScriptsWith(context);
+
+      var worker = LMLayerWorker.install(context);
+      //var worker = new LMLayerWorker({ postMessage: fakePostMessage });
```

Comment:
Delete the comment, and perhaps change the `context` with a single object literal.